### PR TITLE
Issue #39. Added carousel

### PR DESCRIPTION
**Pull request by Vansh-Baghel**
* Added responsive carousel.
* Showing multiple NFTs for different screen sizes.
* Good draggable carousel, with customized design.
nft

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,9 @@
     "react": "^17.0.2",
     "react-color-extractor": "^1.1.2",
     "react-dom": "^17.0.2",
+    "react-elastic-carousel": "^0.11.5",
     "react-icons": "^4.3.1",
+    "react-multi-carousel": "^2.8.2",
```
Comment (yessGlory17) [inline package.json:16]: Is this package used anywhere?

File: src/components/CardList.js [hunk 2]
```diff
@@ -2,15 +2,47 @@ import React from "react";
...
+    >
+      <Carousel
+        ref={carouselRef}
+        enableAutoPlay={true}
+        autoPlaySpeed={4000}
+        isRTL={false}
+        breakPoints={breakPoints}
+        onNextEnd={({ index }) => {
+          clearTimeout(resetTimeout);
+          resetTimeout = setTimeout(() => {
```
Comment (yessGlory17) [inline src/components/CardList.js:33]: Why did you use setTimeout?

Comment (Vansh-Baghel) [reply src/components/CardList.js:33]: > Why did you use setTimeout?

I've set it for infinite loop, so the timeout is to change the nfts per second. The time is in the millisecond, therefore 4000 means 4 sec

File: package.json (hunk 1 above)
Comment (Vansh-Baghel) [reply package.json:16]: No, you can remove _**react-multi-carousel**_ by npm uninstall react-multi-carousel. I tried it but removed it later on.

**Comment by Vansh-Baghel** [comment]
@yessGlory17 Is your doubt resolved? want any changes?

**Comment by yessGlory17** [comment]
@all-contributors please add @Vansh-Baghel for Code

**Pull request merged**